api/tests/integration/tests/formats/cdxml_to_mol.py

```diff
@@ -0,0 +1,23 @@
+
+import sys
+sys.path.append('../../common')
+from env_indigo import *
+
+indigo = Indigo()
+indigo.setOption("molfile-saving-skip-date", True)
+
+print("*** CDXML to mol ***")
+
+root = joinPathPy("molecules/cdxml2", __file__)
+files = os.listdir(root)
+files.sort()
+for filename in files:
+    print(filename)
+    try:
+        mol = indigo.loadMoleculeFromFile(os.path.join(root, filename))
+        print(mol.molfile())
+    except IndigoException as e:
+        print(getIndigoExceptionText(e))
+        print("*** Try as Query ***")
+        mol = indigo.loadQueryMoleculeFromFile(os.path.join(root, filename))
```

Comment:
Why not to catch and print exception here as well? Even caught the exception can fail the test.
In addition, maybe it makes sense to complete the cycle by files to get whole diagnostics?